src/Widgets/index.ts

```diff
@@ -2,6 +2,7 @@ import { initNeoletterFormWidgets } from 'scrivito-neoletter-form-widgets'
 import.meta.glob(['./**/*WidgetClass.ts', './**/*WidgetComponent.tsx'], {
   eager: true,
 })
+import './migrateAttributeNameToData'
```

Comment:
* [ ] FYI Not reviewing. If there are any problems with the migration they can be fixed without touching code. I'd prefer to not have this in the history.